icubam/db/test_wipe.py

```diff
@@ -0,0 +1,33 @@
+from absl.testing import absltest
+from itertools import repeat
+from sqlalchemy import create_engine
+from icubam.db.fake import populate_store_fake
+from icubam.db.wipe import wipe_db
+from icubam.db.store import Store, StoreFactory
+
+class WipeTest(absltest.TestCase):
+    def setUp(self):
+      self.engine = create_engine("sqlite:///:memory:", echo=False)
+      store_factory = StoreFactory(self.engine)
+      self.store = store_factory.create()
+      populate_store_fake(self.store)
+
+
+    def test_wipe(self):
+        with self.engine.connect() as conn:
+            wipe_db(conn.connection, keep_beds=False)
+            users = list(self.store.get_users())
+            self.assertItemsEqual([user.name for user in users],
+                                  repeat("Jean Dumont", len(users)))
```

Comment:
i d rather keep the module, i think it s easier to understand when the function comes from: `itertools.repeat(...`

Reply:
Yup seems clearer that way, thanks